### ISSUE TITLE
A commit containing a line break in its message causes a wrong parse of contents

### ISSUE THREAD

**Issue by Ivan-Perez** [seymen/git-last-commit#8]
Hi!

This module isn't working properly if the commit message contains line breaks. This is what I get when the message body for the last commit is the following:

```
Envío de vídeos al servidor

- API para recibir solicitud de envíos.
- Script para enviar los archivos (php).
```

```javascript
{ shortHash: '7fac2b2',
  hash: '7fac2b2f2c117e4d66c0c159f0bfadeffa7cf75e',
  subject: 'Envío de vídeos al servidor',
  sanitizedSubject: 'Env-o-de-v-deos-al-servidor',
  body: '- API para recibir solicitud de envíos.',
  authoredOn: '- Script para enviar los archivos (php).',
  committedOn: '',
  author: { name: '1506077654', email: '1506077654' },
  committer: { name: 'Iván Pérez', email: 'ivan@*****.com' },
  notes: 'Iván Pérez',
  branch: 'ivan@*****.com',
  tags: [] } 
```

Everything is messed up. Is there any workaround to avoid that? Thanks!!

**Comment by klaudiomilankovic** [comment]
I have a fix for this.  On line 29 replace command with this. 

var command = getCommand(splitCharacter);

function getCommand(splitCharacter) {
    return 'git log -1 --pretty=format:"' + prettyFormat.join(splitCharacter) + '"' +
        ' && git rev-parse --abbrev-ref HEAD' +
        ' && git tag --contains HEAD';
}

**Comment by zhangyuheng** [comment]
Maybe you could try this package: https://www.npmjs.com/package/last-commit-log, It works for me.

**Comment by seymen** [comment]
Fixed by #10
